modelhub/tests_modelhub/functional/modelhub/test_model_hub_logistic_regression.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright 2021 Objectiv B.V.
+"""
+
+# Any import from modelhub initializes all the types, do not remove
+
+
+import pytest
+from tests.functional.bach.test_data_and_utils import get_df_with_test_data
+from tests_modelhub.functional.modelhub.logistic_regression_test_utils import TestLR
+
+
+def test_fitted_model(engine):
```

Comment:
engine fixture doesn't exist in modelhub pytest session, it only works for bach. so, you have to use `db_params` fixture instead and call `create_engine_from_db_params` (https://github.com/objectiv/objectiv-analytics/blob/263f4fefebc172ff3891422fb8a8f2dd23cb69a3/modelhub/tests_modelhub/data_and_utils/utils.py#L77). But this still hasn't been merged tho. It's on https://github.com/objectiv/objectiv-analytics/pull/788
```suggestion
from tests_modelhub.data_and_utils.utils import create_engine_from_db_params
def test_fitted_model(db_params):
    engine = create_engine_from_db_params(db_params)
```